go.work

```diff
@@ -2,78 +2,80 @@ go 1.19
 
 use (
 	.
+	./plugins/amqp
 
 	// define the plugins
 	./plugins/core
 	./plugins/dubbo
+	./plugins/echov4
```

Comment:
Please don't change the original workspace, just add your project into the correct place.